corehq/apps/app_manager/management/commands/find_broken_builds.py

```diff
@@ -133,7 +146,7 @@ def get_build_ids(start, end):
     return builds_ids
 
 
-def find_broken_builds(checker, builds_ids):
+def find_broken_builds(checker, builds_ids, reason=None):
     for build in iter_docs(Application.get_db(), builds_ids):
         error = checker(build)
         if error:
```

Comment:
Will this always evaluate to true if `all_builds` is passed as the check function since `all_builds` always returns the `error`?

Reply:
Oh, I may have misunderstood the purpose of this change. The checker is intended to be used to find all builds that are built within a time frame or be a certain build_id such that it can be flagged, so this is what we want.

What confused me is that in the general case, the returned results won't all be broken builds, which conflicts with the function name `find_broken_builds`